src/devices/danfoss.ts

```diff
@@ -350,6 +348,127 @@ const definitions: Definition[] = [
                 'danfossMultimasterRole'], options);
         },
     },
+    {
+        zigbeeModel: ['0x0210', '0x8040', '0x8041', '0x8042'],
```

Comment:
Please use the `fingerprint` (same as for the `Icon` where you copied it from)

Reply:
Hi, thanks for suggestions, I have implemented them now.